lib-ee/emqx_ee_bridge/src/kafka/emqx_bridge_impl_kafka_consumer.erl

```diff
@@ -145,6 +145,16 @@ on_start(InstanceId, Config) ->
                 instance_id => InstanceId,
                 kafka_hosts => BootstrapHosts
             });
+        {error, already_present = Reason} ->
+            ?SLOG(error, #{
+                msg => "failed_to_start_kafka_consumer_client",
+                instance_id => InstanceId,
+                kafka_hosts => BootstrapHosts,
+                reason => emqx_misc:redact(Reason)
+            }),
+            throw(
```

Comment:
do we really need to throw here? if the client is `already_present`,
we should probably stop it and restart (given the config might have changed)?
otherwise if config is not changed, we can just return ok ?

Reply:
When the user clicks "Reconnect" in the UI, there's no config change, just `emqx_bridge_resource:start/2` is called without any changes to the existing config.

On the one hand, I guess we could indeed just return ok, the status will stay "Disconnected", and the user will have to investigate the logs. 

On the other hand, `already_present` means that the client is in the supervision tree but not running, so it might not be exactly a successful start. :thinking:

Reply:
A similar issue was raised by QA for the Kafka Producer bridge.

At this point, the returned error offers no hint about what went wrong, but the user can inspect the logs and check that the password is wrong, or the connection cannot be established. 

How about we return a hint for the user to check the logs?  Like `"Failed to start Kafka client. Please inspect the EMQX logs and your connection parameters."`